.github/PULL_REQUEST_TEMPLATE/new-pep.md

```diff
@@ -0,0 +1,72 @@
+---
+name: New PEP
+about: Submit a new draft PEP for consideration
+labels: "new-pep"
+---
+
+<!--
+
+Please read and follow PEP 1 and PEP 12 before submitting a new PEP:
+
+https://peps.python.org/pep-0001/#submitting-a-pep
+
+https://peps.python.org/pep-0012/#how-to-use-this-template
+
+Make sure to include the PEP number in the pull request title; for example:
+
+PEP 9999: <Title of PEP>
+
+See our Contributing Guidelines (CONTRIBUTING.rst) for more information.
+
+Thanks!
+
+-->
+
+
+# New PEP
+
+<!--
+
+Briefly describe your new PEP here, and anything else we should know.
+Be sure to link to any previous discussions or issues motivating/drafting it.
+
+-->
+
+
+
+## Checklist
+
+<!--
+
+You should use the following as a guide when double-checking your PEP,
+and you can help complete some of it yourself if you like
+by ticking any boxes you're sure about, like this: [x]
+
+If you're unsure about anything, just leave it blank and we'll take a look.
+
+-->
+
+* [ ] File created from the [latest PEP template](https://github.com/python/peps/blob/main/pep-0012/pep-NNNN.rst?plain=1)
+
+### Content requirements
+
+* [ ] PEP topic [discussed in a suitable venue](https://peps.python.org/pep-0001/#start-with-an-idea-for-python) and general agreement that a PEP is appropriate
+* [ ] Content is reasonably sound, complete and makes at least basic technical sense
+* [ ] Abstract and Copyright sections completed, with the required wording in the latter
+* [ ] [Suggested sections](https://peps.python.org/pep-0012/#suggested-sections) included (unless not applicable)
```

Comment:
Yeah, I'd certainly find that useful as both a PEP author and reviewer, I was just worried that it would make it too long, but I've gone and and implemented it in another comment